File: java/oop/InterfaceApp/src/com/techlabs/icrudable/InvoiceDB.java
package com.techlabs.icrudable;

public class InvoiceDB implements ICRUDable{
	public void create(){
		System.out.println("InvoiceDB is created");
	}
	public void update(){
		System.out.println("InvoiceDB is updated");
	}
	public void read(){
		System.out.println("InvoiceDB is Showing");
	}
	public void delete(){
		System.out.println("InvoiceDB is deleting");
	}
}

File: dotNet/CSharp/InterFaceCRUDApp/InterFaceCRUDApp/InvoiceDB.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace InterFaceCRUDApp
{
    public class InvoiceDB : ICRUDAble
    {
        public void Create()
        {
            Console.WriteLine("InvoiceDB is Created");
        }

        public void Delete()
        {
            Console.WriteLine("InvoiceDB is Deleting");
        }

        public void Read()
        {
            Console.WriteLine("InvoiceDB is Reading");
        }

        public void Update()
        {
            Console.WriteLine("InvoiceDB is Updating");
        }
    }
}
